test/defs.hpp

```diff
@@ -25,6 +25,7 @@
 #define PATCHES_PATH SMCE_TEST_DIR "/patches/"
 #define STORAGE_PATH SMCE_TEST_DIR "/storage/"
 #define MANIFESTS_PATH SMCE_TEST_DIR "/manifests/"
+#define CACHED_DOWNLOADS_PATH SMCE_PATH "/cached_downloads/"
```

Comment:
It would be very wrong for this to be used in another file; might as well just move it to the only file it'll ever be used

Reply:
moving it to LibManagement and making it a local variable to the test_case